### DISCUSSION TITLE
sa to coco conversion and upgrade to version 3.0

### DISCUSSION THREAD

**Discussion by duby69** [superannotateai/superannotate-python-sdk#10] [Q&A] [answered]
Hi,
I did upgrade to version 3.0 and now conversions from sa to COCO doesn't work.
It say:
Traceback (most recent call last):
  File "convert.py", line 3, in <module>
    sa.export_annotation(
TypeError: export_annotation() takes from 4 to 6 positional arguments but 7 were given.

Is there any documentation regarding arguments?

**Comment by hovnatan** [comment]
Sorry, there was a issue with documentation update after release. I just updated the documentation at https://superannotate.readthedocs.io/en/stable/superannotate.sdk.html. Basically the last argument, platform, was removed from the export_annotation(). Does this answer your question? 
Thanks,

**Reply by duby69** [reply]
Thnx,
Now conversion worked, but at the COCO output side I got only person_keypoints and in the image_set folder only classes_mapper.json, but no images and image__objects.json.
Did also the format of sa project files changed?
I need to create classes folder and put classes.json there, and other files change to annotations.json___objects.json and config.json___objects.json.

**Comment by hovnatan** [comment]
What is your workflow? Do you export in SA format then try to convert to
COCO?

**Reply by duby69** [reply]
Yes, export in sa format and then convert to COCO. It worked well on the previous SDK version. After update to 3.0 version, I got into problems.
I'm using this command:

import superannotate as sa

sa.export_annotation(
    "convert/coco/input/SENDD_P8_1_4_4_ZN",
    "convert/coco/output/SENDD_P8_1_4_4_ZN",
    "COCO", "person_keypoints_SENDD_P8_1_4_4_ZN", "Vector","keypoint_detection"
)

**Comment by hovnatan** [comment]
We have changed our JSON annotation format in this release. Could you please re-export your project and try again?

**Reply by duby69** [reply]
I'm using a desktop version for annotation, is there also a changed JSON annotation format?

**Comment by hovnatan** [comment]
Unfortunately we have not updated the Desktop app yet with the new annotation format. It will be updated by the end of this week. You can downgrade the SDK to older version to work with the data until then. Sorry about that.

**Reply by duby69** [reply]
thnx.
which older version works for sure then?

**Comment by hovnatan** [comment, marked as answer]
2.5.12

**Reply by duby69** [reply]
thnx
